node/utils/flags/isSellerFlag.ts

```diff
@@ -0,0 +1,5 @@
+import { sellerFlag } from "./sellerFlag";
+
+export function isSellerFlagged(sellerName: string) {
```

Comment:
Can you rename this to a more specific name?

`isSellerFlaggedForChangingOnSimulationBasedOnRegion`

I think that this way we can't avoid having to figure out why we're changing the seller.